tests/system/test_active_response/test_netsh_windows_command/test_firewall_alerts/data/test_cases/cases_firewall_alerts.yaml

```diff
@@ -0,0 +1,41 @@
+- name: firewall_disabled
+  description: Check that the firewall is disabled.
```

Comment:
Update this

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3787/commits/e4ded61433bf9f440dd1336be139e4ecb01581ec)